bcda/auth/alpha.go

```diff
@@ -54,32 +54,28 @@ func (p AlphaAuthPlugin) DeleteClient(params []byte) error {
 }
 
 // can treat as a no-op or call RequestAccessToken
-func (p AlphaAuthPlugin) GenerateClientCredentials(params []byte) ([]byte, error) {
-	clientID, err := GetParamString(params, "clientID")
-	if err != nil {
-		return nil, err
-	}
-
+func (p AlphaAuthPlugin) GenerateClientCredentials(clientID string) (Credentials, error) {
 	aco, err := getACOFromDB(clientID)
 	if err != nil {
-		return nil, fmt.Errorf(`no ACO found for client ID %s because %s`, clientID, err)
+		return Credentials{}, fmt.Errorf(`no ACO found for client ID %s because %s`, clientID, err)
 	}
 
 	if aco.ClientID == "" {
-		return nil, fmt.Errorf("ACO %s does not have a registered client", clientID)
+		return Credentials{}, fmt.Errorf("ACO %s does not have a registered client", clientID)
 	}
 
 	err = p.RevokeClientCredentials([]byte(fmt.Sprintf(`{"clientID":"%s"}`, clientID)))
 	if err != nil {
-		return nil, fmt.Errorf("unable to revoke existing credentials for ACO %s because %s", clientID, err)
+		return Credentials{}, fmt.Errorf("unable to revoke existing credentials for ACO %s because %s", clientID, err)
 	}
 
+	params := []byte(fmt.Sprintf(`{"clientID":"%s", "ttl":%d}`, clientID, 1))
```

Comment:
I put a magic number here...

Reply:
can't do this; it will create an access token lasting 1 hour in alpha.